portafly/src/i18n/locales/en/shared.json

```diff
@@ -0,0 +1,30 @@
+{
+  "logo_alt_text": "Red Hat Integration logo",
+  "logo_alt_text_desc": "Alt attribute of the main logo",
+  "nav_items": {
```

Comment:
These nested sections, and all those quotes are so annoying when working with JSON. YAML should be the format as we are using in Rails. I'm shooting a PR to tackle this.